pkg/transfers/router.go

```diff
@@ -57,11 +57,10 @@ func NewRouter(
 ) *Router {
 	limitChecker, err := limiter.New(cfg.Transfers.Limits)
 	if err != nil {
-		err = fmt.Errorf("problem creating transfer limiter: %v", err)
-		cfg.Logger.Log("transfers", err)
-		panic(err.Error())
+		err = cfg.Logger.LogErrorf("problem creating transfer limiter: %v", err).Err()
+		panic(err)
 	}
-	cfg.Logger.Log("transfers", fmt.Sprintf("setup %T limit checker", limitChecker))
+	cfg.Logger.Logf("setup %T limit checker", limitChecker)
 	return &Router{
 		Logger:    cfg.Logger,
```

Comment:
what do you think about adding more context to logger like this:
```go
Logger:    cfg.Logger.Set("package", "transfers"),
```

Reply:
I see it's useless as passed instance of logger will not be use inside router (at least I don't see where it's used).